robots/cmd/flake-report-creator/BUILD.bazel

```diff
@@ -6,11 +6,29 @@ go_library(
     importpath = "kubevirt.io/project-infra/robots/cmd/flake-report-creator",
     visibility = ["//visibility:private"],
     deps = [
-        "//robots/pkg/flakefinder:go_default_library",
-        "@com_google_cloud_go_storage//:go_default_library",
+        "//robots/cmd/flake-report-creator/cmd:go_default_library",
+        "@com_github_sirupsen_logrus//:go_default_library",
     ],
 )
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "app",
+    embed = [":go_default_library"],
```

Comment:
If we find no proper solution regarding to the ca certificates, we can use a normal base image and just add the binary:


```
container_image(
    name = "app",
    base = "@myimage//image",
    embed = [":go_default_library"]
)
```